website/src/components/Submission/FileUpload/UploadComponent.tsx

```diff
@@ -19,6 +19,8 @@ export const UploadComponent = ({
     const [isDragOver, setIsDragOver] = useState(false);
     const isClient = useClientFlag();
 
+    // TODO It feels like it would be useful to return a ProcessedFile here, so we know if it's compressed or not etc.
```

Comment:
can this TODO be removed now?

Reply:
There seems to be some issues with uploading .gz files with these changes (.gz files are still working on main): 
![image](https://github.com/user-attachments/assets/bf31945a-d66b-48fe-ad33-69503bca82df)